tox.ini

```diff
@@ -1,14 +1,17 @@
 [tox]
 envlist = packaging, pep8, black, py37, py38, py39, py310, pypy3
+isolated_build = True
 
-[testenv]
+[testenv:py]
 deps =
     coverage
     nose2
+    pytest
 
 commands =
     coverage run --source canonicaljson -m nose2
     coverage report -m --fail-under 100
+    pytest tests
```

Comment:
Why are we using two different test runners (pytest and nose2) here?

Reply:
My mistake, I went over this quickly while trying to fix other things and hadn't realised nose2 was a test runner.